osx/spec/localhost/vim_spec.rb

```diff
@@ -0,0 +1,7 @@
+require 'spec_helper'
+
+describe 'vim' do
+  describe package('vim') do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.